add_up.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+def add_up(i)
+  if i == 1
+    1
+  else
+    i + add_up(i - 1)
+  end
+end
+
+# call the function 3 times using while loop
+
+def print_three_times(number)
```

Comment:
Your `print_three_times` methods look great! However, I believe the instructions were asking for you to call the `add_up` function three times, and have each time return the value.

For example, call the function like `add_up(3)` and have this function return `6`, as 1+2+3 = 6. Then, call the function another two times with a different argument and have it print the return value.